apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorProgressReporter.java

```diff
@@ -111,34 +68,6 @@ long getPid() {
     return ProcessHandle.current().pid();
   }
 
-  /**
-   * Does the conversion and wraps any checked exception in an unchecked one.
-   *
-   * @param message object to convert into JSON
-   * @return converted JSON string
-   */
-  static String convertMessageToJson(SqsProgressMessage message) {
-    try {
-      return objectMapper.writeValueAsString(message);
-    } catch (IOException ex) {
-      throw new UncheckedIOException(ex);
-    }
-  }
-
-  /**
-   * Does the conversion and wraps any checked exception in an unchecked one.
-   *
-   * @param jsonMessage JSON string representation of a {@link SqsProgressMessage}
-   * @return converted {@link SqsProgressMessage}
-   */
-  static SqsProgressMessage convertJsonToMessage(String jsonMessage) {
-    try {
-      return objectMapper.readValue(jsonMessage, SqsProgressMessage.class);
-    } catch (IOException ex) {
-      throw new UncheckedIOException(ex);
-    }
-  }
-
```

Comment:
All of this jackson specific boilerplate has been moved into `JacksonJsonConverter` which is an implementation of `JsonConverter`.  Over time we can migrate other code to use these instead of directly using Jackson.